src/plugins/cosmos/components/StakingButtons/StakingButtons.tsx

```diff
@@ -0,0 +1,14 @@
+import { Flex, FlexProps } from '@chakra-ui/layout'
+import { Button } from '@chakra-ui/react'
+import { Text } from 'components/Text'
+
+export const StakingButtons = (styleProps?: FlexProps) => (
```

Comment:
Note: #1058 will implement action buttons for stake/unstake input. They effectively route to the same routes, so it would be nice to be able to reuse the same component there, with different styling and additional logic for displaying the active button.